data/brands/shop/bakery.json

```diff
@@ -417,7 +417,7 @@
     {
       "displayName": "BreadTalk",
       "id": "breadtalk-96a6b3",
-      "locationSet": {"include": ["sg"]},
+      "locationSet": {"include": ["142", "150"]},
```

Comment:
Note what is actually described there - from https://www.straitstimes.com/business/breadtalk-debuts-in-london-with-din-tai-fung-flagship

> Taiwanese restaurant Din Tai Fung opened in London's Covent Garden yesterday, as the BreadTalk Group marked its first foray into Europe.

BreadTalk Group opened something, that does NOT mean any BreadTalk bakeries are in Europe

Is there any confirmation of even single BreadTalk bakery in Europe? It seems that there are none in Poland, Google Maps also fails to find any.